spec/command/audit_spec.rb

```diff
@@ -312,6 +312,25 @@ def self.it_fails command, *raise_error_args
           expect { invoke }.to write(" created role super:user")
         end
       end
+
+      describe 'audit of ssh:sudo' do
+        let(:ssh_event) { default_audit_event.merge('kind' => 'audit', 'facility' => 'ssh',  'action' => 'sudo', 'command' => '/bin/ls', 'system_user' => 'test_user', 'target_user' => 'root') }
+        context 'when sudo successful' do
+          let(:test_event) { ssh_event.merge('allowed' => true) }
+          it 'prints <user> ran <command> for successful command' do
+            expect { invoke }.to write(" test_user ran '/bin/ls' as root")
+          end
+        end
+
+        context 'when sudo fails' do
+          let(:test_event) { ssh_event.merge('allowed' => false) }
+          
+          it 'prints <user> attempted to run <command> for successful command' do
```

Comment:
`successful`? In general, I think the "for successful command" phrase is redundant with the context.